src/serverFunctions/getTextAndToCase.js

```diff
@@ -0,0 +1,10 @@
+function getTextAndToCase(request) {
+  const queryParams = request.url.split('?');
+  const text = queryParams[0].slice(1);
+  const query = new URLSearchParams(queryParams[1]);
+  const toCase = query.get('toCase');
```

Comment:
you can also use `new URL` to implement this logic, it may look more clean:
```javascript
const normalizedURL = new URL(req.url, `http://${req.headers.host}`);

    const targetCase = normalizedURL.searchParams.get('toCase');
    const originalText = normalizedURL.pathname.slice(1);
```

Reply:
but it's up to you